cob_obstacle_distance_moveit/src/cob_obstacle_distance_moveit/obstacle_distance_moveit.cpp

```diff
@@ -169,13 +160,14 @@ void ObstacleDistanceMoveit::calculateDistanceTimerCallback(const ros::TimerEven
         {
             std::string robot_self_name = selfcollision_it->first;
             collision_detection::AllowedCollision::Type type;
-            if(acm.getEntry(robot_link_name, robot_self_name, type))
+
+            if(acm_.getEntry(robot_link_name, robot_self_name, type))
             {
-                if(type == collision_detection::AllowedCollision::NEVER)
+                if(type == collision_detection::AllowedCollision::ALWAYS)
```

Comment:
discussed in person

additional idea:
introduce a link list (auto-generated from URDF) which contains all "dummy" links...for those, distances should not be computed.